detector/wordpress_test.go

```diff
@@ -82,3 +83,161 @@ func TestRemoveInactive(t *testing.T) {
 		}
 	}
 }
+
+// ref: https://wpscan.com/docs/api/v3/v3.yml/
+func Test_convertToVinfos(t *testing.T) {
```

Comment:
If you also prepare a pattern with cve, it would be nice to be able to cover it.

Reply:
I understand!